queue_services/entity-emailer/src/entity_emailer/email_processors/involuntary_dissolution_stage_1_notification.py

```diff
@@ -0,0 +1,111 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Email processing rules and actions for involuntary_dissolution stage 1 overdue ARs notifications."""
+from __future__ import annotations
+
+from pathlib import Path
+
+from entity_queue_common.service_utils import logger
+from flask import current_app
+from jinja2 import Template
+from legal_api.models import Business, Furnishing
+
+from entity_emailer.email_processors import get_entity_dashboard_url, get_jurisdictions, substitute_template_parts
+
+
+PROCESSABLE_FURNISHING_NAMES = [
+    Furnishing.FurnishingName.DISSOLUTION_COMMENCEMENT_NO_AR.name,
+    Furnishing.FurnishingName.DISSOLUTION_COMMENCEMENT_NO_TR.name,
+    Furnishing.FurnishingName.DISSOLUTION_COMMENCEMENT_NO_AR_XPRO.name,
+    Furnishing.FurnishingName.DISSOLUTION_COMMENCEMENT_NO_TR_XPRO.name
+]
+
+PROCESSABLE_LEGAL_TYPES = [
+    Business.LegalTypes.COMP,
+    Business.LegalTypes.BC_ULC_COMPANY,
+    Business.LegalTypes.BC_CCC,
+    Business.LegalTypes.BCOMP
+]
```

Comment:
can remove this.  no need to check the legal type in emailer

Reply:
Removed